test/fail_compilation/checkimports1b.d

```diff
@@ -3,7 +3,7 @@
 TEST_OUTPUT:
 ---
 fail_compilation/checkimports1b.d(16): Deprecation: local import search method found struct `imports.diag12598a.lines` instead of variable `checkimports1b.C.lines`
-fail_compilation/checkimports1b.d(16): Error: struct `lines` is a type, not an lvalue
+fail_compilation/checkimports1b.d(16): Error: cannot modify `lines`. It is a struct
```

Comment:
Maybe:
```
`lines` is a struct definition and cannot be modified 
```

Basically holding the information that it's a type / definition makes this message clearer IMHO.